ec2-consistent-snapshot

```diff
@@ -87,7 +87,7 @@ sync
 
   # Find the mounted EXT4 and XFS partitions and freeze them
   # To avoid hanging the machine in unrecoverable way, don't freeze "/".
-  for target in $(findmnt -nlo TARGET -t ext4,xfs); {
+  for target in $(findmnt -nlo UUID,TARGET -t ext4,xfs  | awk '!a[$1]++ { print $2 }'); {
```

Comment:
Seems fine `awk` is available pretty much everywhere, but I bother to list the external binaries we depend on in the README: https://github.com/RideAmigosCorp/ec2-consistent-snapshot.sh/issues/1